src/test-data/ModificationsData.js

```diff
@@ -42,7 +42,7 @@ export const exampleModificationAltersTopLevel = [
     "control-id": "control-1",
     adds: [
       {
-        "by-id": "control-1",
+        "by-id": "control-1_smt",
```

Comment:
This object shouldn't be changed. We've been using the term "top-level" to refer to modifications that apply to the control as a whole, not any of the individual control parts.

Changing this data also changes the Storybook; I believe the intention with the modifications in the Storybook page for the Control component was to show off the top-level modifications and how to render them; this change gets rid of that.

Reply:
That makes sense, will make those changes. I will create another test in `OSCALControl.test` and the appropriate test data in `ModificationsData`